### DISCUSSION TITLE
Add new user to conversation using chat service

### DISCUSSION THREAD

**Discussion by jh-skaion** [facebookresearch/ParlAI#3640] [General]
Hello @stephenroller, i have been trying to add a new user to a conversation,which was already started between human and bot, using the example given with websocket and terminal. 
i have been doing a lot of reading of the chat service manual to see how it could be modified to accept new user but i have not been able to figure out. 
Do you or anyone from your team can give me some guidance or point me in the right direction?
thank you for your time

**Comment by stephenroller** [comment]
@klshuster are you able to make some recommendations?

**Reply by klshuster** [reply]
i would need some more information - what do you mean by adding a new user? do you mean having a separate conversation occurring between a human and a bot? or having a three-way conversation?

**Reply by jh-skaion** [reply]
hello sorry for the delay... i was referring to a three or more ways... but we kinda working on work around... the problem i am having right now is exposing the websocket to my local network so i can connect from other computers. 

@klshuster Do you have a guide or something for me to look how it is done?
